airbyte-integrations/connectors/source-marketo/unit_tests/test_source.py

```diff
@@ -314,3 +314,45 @@ def test_get_updated_state(config, latest_record, current_state, expected_state)
     if expected_state == "start_date":
         expected_state = {"updatedAt": config["start_date"]}
     assert stream.get_updated_state(latest_record, current_state) == expected_state
+
+
+def test_filter_null_bytes(config):
+    stream = Leads(config)
+
+    def to_iterable(lines):
+        for line in lines:
+            yield line
+    test_lines = [
+        "Hello\x00World\n",
+        "Name,Email\n",
+        "John\x00Doe,john.doe@example.com\n"
+    ]
+    expected_lines = [
+        "HelloWorld\n",
+        "Name,Email\n",
+        "JohnDoe,john.doe@example.com\n"
+    ]
+    filtered_lines = stream.filter_null_bytes(to_iterable(test_lines))
```

Comment:
Please delete `to_iterable` function and just use `filtered_lines = stream.filter_null_bytes(test_lines)` instead as `test_lines` is already an iterable. Thanks!

Reply:
Sure! 
Done. Please have a look.

Reply:
Thank you!